docs/gen-ai/llm-metrics.md

```diff
@@ -0,0 +1,128 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: LLM metrics
```

Comment:
I think we've switched to GenAI everywhere including the attribute/metrics, let's do it for titles in this doc as well?

Reply:
llm-spans still has the header LLM spans.

Reply:
you're right, but it's about to be changed in #917

Reply:
I updated the title for this and change the file name to gen-ai-metrics